client/src/views/OrganizationPage.vue

```diff
@@ -43,6 +43,6 @@
 <script setup lang="ts">
 import {
   IonSegment, IonSegmentButton, IonToolbar,
-  IonPage, IonButtons, IonContent, IonRouterOutlet, IonHeader, IonTitle, IonLabel, IonBackButton
+  IonPage, IonButtons, IonContent, IonRouterOutlet, IonHeader, IonTitle, IonLabel, IonBackButton,
```

Comment:
we don't want inline ones we said no?

Reply:
It's not inline because the `}` is on another line, it's a weird case that the rule does not take into account I think.

Reply:
yeah, it's ugly, maybe split as one per line?
Maybe we could take a look at this rule to address this?
https://eslint.org/docs/latest/rules/object-curly-newline

Reply:
It's probably hard to enforce it well with a rule.
`import { ref, computed } from 'vue';` is perfectly fine, there's no need to add a new line or anything. But when the line starts becoming very long, it's probably better to split it and put one element per line. The rule does not really have something to solve this. Maybe `minProperties` but then again, if the properties are small it's not much of a problem.

Reply:
In those specific cases, I put one element per line. I played a bit with `object-curly-newline` but could not find something satisfiying to deal with this issue.